pulsar/internal/http_client.go

```diff
@@ -52,7 +52,7 @@ func (c *httpClient) Close() {
 }
 
 type HTTPClient interface {
-	Get(endpoint string, obj interface{}) error
+	Get(endpoint string, obj interface{}, params map[string]string) error
```

Comment:
Is this change related to the certificate updates

Reply:
> Is this change related to the certificate updates

it is related to Pulsar 2.8.0, seems the `GetPartitionedTopicMetadata` with http lookup service has to pass `checkAllowAutoCreation=true` as params to get the partition num of the non-existed topic. So once we run CI with Pulsar 2.8.0, the tests will fail. [related with https://github.com/apache/pulsar/pull/10601]